doc-content/enterprise-only/getting-started-kogito/con-kogito-microservices-on-ocp.adoc

```diff
@@ -0,0 +1,8 @@
+[id="con-kogito-microservices-on-ocp_{context}"]
+= {KOGITO} on {OPENSHIFT}
+
+You can deploy {KOGITO} microservices on {OPENSHIFT} for cloud implementation. In this architecture, {KOGITO} microservices are deployed as OpenShift pods that you can scale up and down individually to provide as few or as many containers as required for a particular service. You can use standard OpenShift methods to manage the pods and balance the load.
```

Comment:
`You can use standard OpenShift methods to manage the pods and balance the load.`
This sentence can be misleading as user have to interact with KogitoRuntime CustomResource to define requirements, Kogito operator will then adjust the OpenShift pods and resources accordingly. Maybe would be better to drop this sentence?

Reply:
Done, removed the sentence.